install.cmd

```diff
@@ -8,25 +8,19 @@ IF NOT "%~1"=="" SET PREFIX="%~1"
 SET HTMLDIR=%PREFIX%\share\doc\git-doc
 SET GITEXTRAS=%~dp0
 
-IF NOT EXIST %PREFIX%\bin MKDIR %PREFIX%\bin
-
-SET COMMANDS_WITHOUT_REPO=git-alias git-extras git-fork git-setup
+IF NOT EXIST %PREFIX%\etc\bash_completion.d MKDIR %PREFIX%\etc\bash_completion.d
+COPY %GITEXTRAS%\etc\bash_completion.sh	%PREFIX%\etc\bash_completion.d\git-extras
 
 FOR /R %GITEXTRAS%\bin %%i in (*.*) DO (
 	ECHO #!/usr/bin/env bash > %PREFIX%\bin\%%~ni
 	TYPE %GITEXTRAS%\helper\reset-env >> %PREFIX%\bin\%%~ni
 	TYPE %GITEXTRAS%\helper\git-extra-utility >> %PREFIX%\bin\%%~ni
-	TYPE %GITEXTRAS%\helper\is-git-repo >> %PREFIX%\bin\%%~ni
+	>NUL FINDSTR /C:%%~ni %GITEXTRAS%\need_git_repo || (
+		TYPE %GITEXTRAS%\helper\is-git-repo >> %PREFIX%\bin\%%~ni
```

Comment:
It took me a while to understand what this does:

```
 +     >NUL FINDSTR /C:%%~ni %GITEXTRAS%\need_git_repo || (
 +		TYPE %GITEXTRAS%\helper\is-git-repo >> %PREFIX%\bin\%%~ni
 +	)
``` 

`>NUL` prevents output from the findstr. Funny that you can write the pipe before the actual command...

Could probably easier to understand if this is refactored to 

```
FINDSTR /C:%%~ni %GITEXTRAS%\need_git_repo >NUL 
if %errorlevel% == 0 (...)
```